### ISSUE TITLE
[remux] Enabling Remux blocks 1st-party cloud sync

### ISSUE THREAD

**Issue by RedshiftVelocities** [rmkit-dev/rmkit#183]
Ever since update 2.15.1.1189, after I run `systemctl enable --now remux` and xochitl restarts, the first party cloud sync on the device breaks. I can create and edit documents on the device just fine, but the changes won't show up in the desktop app. I also am unable to load files onto the device from the desktop app with remux enabled. Running `systemctl disable --now remux` and then `systemctl enable --now remux` solves the problem, but then obviously I can't switch apps. I am on the Remarkable 1, and using Toltec to install remux.

**Comment by Eeems** [comment]
You might want to include some log output to help investigate. Cloud sync would not be running while you are switched away from xochitl. Xochitl will only trigger a document sync after you go to the main screen after editing a file. At that point in time, you still have to wait for the sync to finish before you can switch applications.

**Comment by RedshiftVelocities** [comment]
Ok, What logs should I include, and where should I go to find them? This problem occurs even when I'm in Xochitl with Remux running in the background. Stopping Remux and running with _just_ Xochitl solves the problem.

**Comment by Eeems** [comment]
The xochitl logs `journalctl -fau xochitl` will follow the logs as they happen.
On the bottom left, do you see a cloud with an X on it? Have you gone into settings and pressed the "check sync" button yet?

**Comment by RedshiftVelocities** [comment]
I hope these are the logs you're looking for.
`reMarkable: ~/ journalctl -fau xochitl
-- Logs begin at Fri 2022-11-04 00:07:42 UTC. --
Nov 06 04:26:58 reMarkable xochitl[160]: 04:26:58.619 rm.wifi                  Scanning: true
Nov 06 04:26:59 reMarkable xochitl[160]: 04:26:59.654 rm.wifi                  wlan0: Address added: CENSORED
Nov 06 04:27:02 reMarkable xochitl[160]: 04:27:02.728 rm.wifi                  Scanning: false
Nov 06 04:27:03 reMarkable xochitl[160]: 04:27:03.552 rm.wifi                  wlan0: Address added: 192.168.1.129
Nov 06 04:27:03 reMarkable xochitl[160]: 04:27:03.569 rm.wifi                  Got gateway: 192.168.1.128
Nov 06 04:27:03 reMarkable xochitl[160]: 04:27:03.589 rm.wifi                  We probably have Internet
Nov 06 04:27:03 reMarkable xochitl[160]: 04:27:03.626 rm.updates               Request finished. Was dispatched at: 2022-11-06T04:26:43.399
Nov 06 04:27:03 reMarkable xochitl[160]: 04:27:03.627 rm.updates               Network error: QNetworkReply::HostNotFoundError Payload: ""
Nov 06 04:27:03 reMarkable xochitl[160]: 04:27:03.628 rm.telemetry
Nov 06 04:27:05 reMarkable xochitl[160]: 04:27:05.094 rm.wifi                  wlan0: Address added: CENSORED


^C
reMarkable: ~/ systemctl enable --now remux
Created symlink /etc/systemd/system/multi-user.target.wants/remux.service → /lib/systemd/system/remux.service.
reMarkable: ~/ journalctl -fau xochitl
-- Logs begin at Fri 2022-11-04 00:07:42 UTC. --
Nov 06 04:27:03 reMarkable xochitl[160]: 04:27:03.569 rm.wifi                  Got gateway: 192.168.1.128
Nov 06 04:27:03 reMarkable xochitl[160]: 04:27:03.589 rm.wifi                  We probably have Internet
Nov 06 04:27:03 reMarkable xochitl[160]: 04:27:03.626 rm.updates               Request finished. Was dispatched at: 2022-11-06T04:26:43.399
Nov 06 04:27:03 reMarkable xochitl[160]: 04:27:03.627 rm.updates               Network error: QNetworkReply::HostNotFoundError Payload: ""
Nov 06 04:27:03 reMarkable xochitl[160]: 04:27:03.628 rm.telemetry
Nov 06 04:27:05 reMarkable xochitl[160]: 04:27:05.094 rm.wifi                  wlan0: Address added: CENSORED
Nov 06 04:29:35 reMarkable xochitl[160]: 04:29:35.376 rm.network.document.sync unregistered no.remarkable.sync
Nov 06 04:29:35 reMarkable systemd[1]: Stopping reMarkable main application...
Nov 06 04:29:35 reMarkable systemd[1]: xochitl.service: Succeeded.
Nov 06 04:29:35 reMarkable systemd[1]: Stopped reMarkable main application.
`

There is no cloud with an X on it on the bottom left, and "check sync" comes back as successful. Nevertheless, the device doesn't sync.

**Comment by stehji** [comment]
I can confirm this issue. For me it is the same since version 1.15. came out. 
Unfortunately i didn't find anything coresponding to this in the log, so I can't bring here any new information. 
It have probably something to do with the new way how sync work on 1.15 (as mentioned in changelog) but I dont know why.
The tablet itself don't show any errors with synchronization in xochitl, but also don´t start synchronization, after notebook is closed. Disabling remux solve this isue. So it is probrably causing this.

**Comment by Eeems** [comment]
@RedshiftVelocities @stehji what does `systemctl status sync.service` output? Based on recent investigation I did to resolve some issues with xochitl starting when it shouldn't in https://github.com/toltec-dev/toltec/pull/641 I think that `sync.service` is just not starting properly when using remux. This in theory should be fixed in the next toltec stable merge (sometime this morning).

**Comment by stehji** [comment]
@Eeems you are right. The sync.service doesn't start when remux does and stay unloaded. `stemctl start sync.service` solves the problem. 
I will check the next merge, when it will  be released, if the problem was solved.

**Comment by stehji** [comment]
I can confirm that everything works right. This issue can be closed.

**Comment by raisjn** [comment]
thanks @Eeems!!
